Makefile

```diff
@@ -3,9 +3,7 @@ GOFLAGS := -v
 PKG := ./...
 TESTS := ".*"
 GOIMPORTS := goimports
-PROGNAME := biscuit
-VERSION := $(shell git describe --long --tags --always)
-GOVERSIONLDFLAG := -ldflags="-X main.Version=$(VERSION)"
+VERSION ?= $(shell git describe --long --tags --always)
 
 .PHONY: build
 build: doc.go test
```

Comment:
To make this more "go like". We may want to have `go generate ./...` create the `doc.go` - but this is totally separate concern.